Salesforce_Advanced/Work_Dot_Com_Command_Center/WellnessForm.xml

```diff
@@ -0,0 +1,623 @@
+									<updates>
+										<update valuesource="fieldvalue" field="PTO_Request" enclosevalueinquotes="true" value="Yes"/>
+									</updates>
+								</action>
+							</iftrueactions>
+						</action>
+						<action type="updateRow" fieldmodel="WellnessAssessment" affectedrows="context">
+							<updates>
+								<update valuesource="fieldvalue" field="Assessment" enclosevalueinquotes="true" value="COVID-19 No Symptoms"/>
+							</updates>
+						</action>
+					</actions>
+					<events>
+						<event>row.updated</event>
+					</events>
+					<fields>
+						<field>Symptoms</field>
+					</fields>
+				</action>
+				<action>
+					<actions>
+						<action type="branch" whenfinished="stop">
+							<formula>{{Contact}}=="Yes"</formula>
+							<iftrueactions>
+								<action type="updateRow" fieldmodel="WellnessAssessment" affectedrows="context">
+									<updates>
+										<update valuesource="fieldvalue" field="Assessment" enclosevalueinquotes="true" value="COVID-19 Exposed"/>
+									</updates>
+								</action>
+							</iftrueactions>
+						</action>
+						<action type="updateRow" fieldmodel="WellnessAssessment" affectedrows="context">
+							<updates>
+								<update valuesource="fieldvalue" field="Assessment" enclosevalueinquotes="true" value="COVID-19 No Symptoms"/>
+							</updates>
+						</action>
+					</actions>
+					<events>
+						<event>row.updated</event>
+					</events>
+					<fields>
+						<field>Contact</field>
+					</fields>
+				</action>
+				<action>
+					<actions>
+						<action type="branch" whenfinished="stop">
+							<formula>{{PTO_Request}}=="Yes"</formula>
+							<iftrueactions>
+								<action type="createRow" model="PTO_Request" appendorprepend="prepend" defaultmodefornewitems="edit" affectedrows="context"/>
+							</iftrueactions>
+						</action>
+						<action type="cancel">
+							<models>
+								<model>PTO_Request</model>
+							</models>
+						</action>
+					</actions>
+					<events>
+						<event>row.updated</event>
+					</events>
+					<fields>
+						<field>PTO_Request</field>
+					</fields>
+				</action>
+			</actions>
+		</model>
+		<model id="Consent" limit="1" query="false" createrowifnonefound="true" datasource="salesforce" sobject="AuthorizationFormConsent">
+			<fields>
+				<field id="ConsentGiverId"/>
+				<field id="ConsentGiver.Name"/>
+				<field id="Status"/>
+				<field id="ConsentCapturedSource"/>
+				<field id="AuthorizationFormTextId"/>
+				<field id="AuthorizationFormText.Name"/>
+			</fields>
+			<conditions>
+				<condition type="modelmerge" value="" field="ConsentGiverId" fieldtargetobjects="Account,Contact,Individual,User" operator="=" model="Individual" enclosevalueinquotes="true" mergefield="Id" novaluebehavior="deactivate"/>
+				<condition type="fieldvalue" value="<<REPLACE WITH AuthorizationFormTextId>>" enclosevalueinquotes="true" field="AuthorizationFormTextId" fieldtargetobjects="AuthorizationFormText"/>
```

Comment:
When I tried to use this XML, Skuid wasn't happy and wouldn't let me save until I removed the brackets here. 

```suggestion
				<condition type="fieldvalue" value="REPLACE WITH AuthorizationFormTextId" enclosevalueinquotes="true" field="AuthorizationFormTextId" fieldtargetobjects="AuthorizationFormText"/>
```

Reply:
Thanks Cody - I've put these into place.